### PR TITLE
Rc 0.11.0

### PR DESCRIPTION
**Pull request by rhornung67**
# Summary

- This PR is to merge the v0.11.0 RC branch into develop prior to the new release.
- It bumps the version number, updates release notes, user docs, updates LC build scripts, includes some minor code cleanup, etc.

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -7,6 +7,59 @@
...
+
+This release contains new features, several notable changes, and some bug fixes.
+
+Notable changes include:
+
+  * New features:
+      * HIP compiler back-end added to support AMD GPUs. Usage is essentially
+        the same as for CUDA. Note that this feature is considered a
+        work-in-progress and not yet production ready. It is undocumented,
+        but noted here, for friendly users who would like to it out.
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md]:
```suggestion
        but noted here, for friendly users who would like to try it out. 
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -7,6 +7,59 @@
...
+        work-in-progress and not yet production ready. It is undocumented,
+        but noted here, for friendly users who would like to it out. 
+      * Updated version of camp third-party library, which includes variety
+        of portability fixes. Most users should not need to concern 
+        themselves with the details of camp.
+      * Added new tutorial material and exercises.
+      * Documentation improvements.
+
+  * API Changes:
+      * RAJA no longer has an optional configuration dependency on CHAI. Users
```
Comment (davidbeckingsale) [inline RELEASE_NOTES.md]: I thought this was included in 0.10.0?

File: RELEASE_NOTES.md (hunk 1 above)
Comment (rhornung67) [reply RELEASE_NOTES.md]: Fixed. Thanks.

File: RELEASE_NOTES.md (hunk 2 above)
Comment (rhornung67) [reply RELEASE_NOTES.md]: You are correct, sir! I removed it.

**Pull request merged**